File: tools/bench/answers/python_haiku/s1/15_wordlens.py
sentence = "pack my box with five dozen liquor jugs"
words = sentence.split()
longest = words[0]
for word in words:
    if len(word) > len(longest):
        longest = word
total_length = 0
for word in words:
    total_length += len(word)
avg_length = total_length / len(words)
print(longest)
print(int(avg_length))

File: tools/bench/answers/rust_haiku/s1/15_wordlens.rs
fn main() {
    let sentence = "pack my box with five dozen liquor jugs";
    let words: Vec<&str> = sentence.split_whitespace().collect();

    let mut longest = words[0];
    for word in words.iter() {
        if word.len() > longest.len() {
            longest = word;
        }
    }

    let mut total_len = 0;
    for word in words.iter() {
        total_len += word.len();
    }

    let avg_len = total_len / words.len();

    println!("{}", longest);
    println!("{}", avg_len);
}
